src/CMakeLists.txt

```diff
@@ -282,6 +333,18 @@ endif()
 string(APPEND LEANC_STATIC_LINKER_FLAGS " ${LEAN_CXX_STDLIB}")
 string(APPEND LEANSHARED_LINKER_FLAGS " ${LEAN_CXX_STDLIB}")
 
+# Link LLVM for code generation
+# TODO: hide this behind a flag
+# Is there a guarantee that the build environment for `leanprover/lean-llvm` will match that of
+# `leanprover/lean4`? Some of the system-libs as for hardcoded paths. For example:
+# 
+# string(APPEND LEANSHARED_LINKER_FLAGS " ${LLVM_CONFIG_LDFLAGS} ${LLVM_CONFIG_LIBS} ${LLVM_CONFIG_SYSTEM_LIBS}")
+# string(APPEND LEANC_STATIC_LINKER_FLAGS " ${LLVM_CONFIG_LDFLAGS} ${LLVM_CONFIG_LIBS} ${LLVM_CONFIG_SYSTEM_LIBS}")
```

Comment:
This commented out stuff can be removed, no?

Reply:
Fixed.